### DISCUSSION TITLE
Autoformatting in None-ls

### DISCUSSION THREAD

**Discussion by mendax1234** [nvimtools/none-ls.nvim#114] [Q&A]
By using none-ls, can I autoformat my code while I writing my code?  Seems that this plugin can only format my code when I use a command to trigger that formatting process right?

**Comment by mochaaP** [comment]
https://github.com/nvimtools/none-ls.nvim/wiki/Formatting-on-save

Notice this article could be a bit out of date, please take it with a grain of salt.

**Reply by mendax1234** [reply]
@mochaaP Thanks for your reply! So, after reading, seems that both the sync formatting and async formatting are doing the format after use save the files. So,  I am wondering whether there is a method to format while I am coding, just like how vscode works (But I am not sure how vscode achieve this)
